### PR TITLE
add go module to version proto

### PR DESCRIPTION
**Pull request by qJkee**
Create a submodule inside proto file directory to be able to fetch it from other projects with go modules

### CODE REVIEW

File: versionpb/hack.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
```
Comment (AlekSi) [inline versionpb/hack.go:1]: I `main` package name can cause problems, and you don't really have to use that name and `func main`.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .vscode
 main
 versionpb/*.go
+!versionpb/hack.go
```
Comment (AlekSi) [inline .gitignore:4]: That could a normal empty non ignored file.

**Comment by AlekSi** [comment]
Ok, but why you don't want to keep generated code in the repository? Typically, it is easier for the user – they can use it directly instead of regenerating it.

**Comment by blez** [comment]
We discussed it and decided that there is no much advance to keep generated client code in this repo. Every user of version service can generate it by itself. Maybe with using different plugins.

**Pull request merged**